### PR TITLE
feat: Add v3 examples

### PR DESCRIPTION
**Pull request by Amzani**
Fixes https://github.com/asyncapi/studio/issues/831

### CODE REVIEW

File: apps/studio/src/examples/websocket-gemini.yml [hunk 1]
```diff
@@ -94,25 +148,29 @@ channels:
...
         $ref: '#/components/schemas/market'
       examples:
         - name: updateMessage
-          summary: Example of an update message that contains a change in price information.
+          summary: >-
+            Example of an update message that contains a change in price
+            information.
           payload:
             type: update
             eventId: 36902233362
```
Comment (KhudaDad414) [inline apps/studio/src/examples/websocket-gemini.yml:176]: `timestamp` and `timestampms` should be string in `date-time` and `time` format accordingly, according the schema mentioned.

File: apps/studio/src/examples/websocket-gemini.yml
Comment (KhudaDad414) [inline apps/studio/src/examples/websocket-gemini.yml:1]: this example is not being validated in the studio.

**Review by KhudaDad414 — CHANGES_REQUESTED** [review]

File: apps/studio/src/examples/websocket-gemini.yml
Comment (Amzani) [reply apps/studio/src/examples/websocket-gemini.yml:1]: Fixed by updating `spec-json-schemas` dependency

**Review by KhudaDad414 — DISMISSED** [review]

**Review by fmvilas — APPROVED** [review]

**Pull request merged**